tests/http/test_07_upload.py

```diff
@@ -522,17 +518,17 @@ def test_07_41_post_urlenc_small(self, env: Env, httpd, nghttpx, proto):
         respdata = open(curl.response_file(0)).readlines()
         assert respdata == indata
 
-    def check_download(self, count, srcfile, curl):
+    def check_download(self, r, count, srcfile, curl):
```

Comment:
All newly-added parameters would be more useful if they had type annotations added as well.

Reply:
Added.